class.jetpack-plan.php

```diff
@@ -9,6 +9,70 @@
  * Provides methods methods for fetching the plan from WordPress.com.
  */
 class Jetpack_Plan {
+	/**
+	 * A cache variable to hold the active plan for the current request.
+	 *
+	 * @var array
+	 */
+	private static $active_plan_cache;
+
+	const PLAN_OPTION = 'jetpack_active_plan';
```

Comment:
I think `const` is PHP 5.3+ only, so we can't use it just yet :(

Reply:
(╯°□°）╯︵ ┻━┻ 

😄

Reply:
It looks like we already use class constants in [many PHP files already](https://github.com/Automattic/jetpack/search?l=PHP&q=const). When I search php class constants, I landed on [this doc](http://php.net/manual/en/language.oop5.constants.php). It looks like the mentions there about 5.3 are more around extra ways to access the class and constant as opposed to support of class constants itself. So, I think we'd be OK.

Reply:
Yup, `const` is fine. Here's a PHP test for it from 5.2.4: https://github.com/php/php-src/blob/8557dabc8a07e543f4b8f3ccd1113072294c459c/tests/classes/constants_scope_001.phpt